webpack.config.babel.js

```diff
@@ -118,10 +119,14 @@ export default {
       data: thisPackage,
     }]}),
     new JSONWebpackPlugin({
-      template: "src/webextension/manifest.json.tpl",
+      template: path.join(__dirname, "src/webextension/manifest.json.tpl"),
       filename: "webextension/manifest.json",
       data: thisPackage,
     }),
+    new DirListWebpackPlugin({
+      directory: path.join(__dirname, "src/webextension/locales"),
+      filename: "webextension/locales/locales.json",
```

Comment:
I'm a bit confused here... So, it seems like we're just grabbing a list of all files/folders from a specified directory and outputting it to a "locales.json" file here. That seems OK and we've done that for other add-ons for development, but usually for production we'd check translation %%s and make sure a locale is 95-100% translated in Pontoon before adding it as an available locale. Otherwise some users may get 15% Russian and then 85% fallback English strings (and a poor user experience, or whatever).

Reply:
Should we just manually update locales.json then? Or maybe just do this automatic thingy for dev builds?

Reply:
I think this is fine for now, but we may need to tweak it once we get closer to a beta release and get wired into pontoon.

Reply:
Well, I added a production config that needs to be manually updated, plus a README explaining how it works for localizers.